src/Modules/Power.asm

```diff
@@ -38,88 +37,86 @@
 ;
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 switch_power_off:
-    All_Pwm_Fets_Off                ; Turn off all pwm FETs
-    All_Com_Fets_Off                ; Turn off all commutation FETs
+    All_Pwm_Fets_Off                              ; Turn off all pwm FETs
+    All_Com_Fets_Off                              ; Turn off all commutation FETs
     Set_All_Pwm_Phases_Off
     ret
 
-
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
-;
-; Set pwm limit low rpm
-;
-; Sets power limit for low rpm
-;
-;**** **** **** **** **** **** **** **** **** **** **** **** ****
+    ;**** **** **** **** **** **** **** **** **** **** **** **** ****
```

Comment:
Is there a way to keep banners without indentation? If not I can move them manually to the left.

Reply:
Yes, if we open a banner with ;**** and close it the same way, I can parse that.